api/src/main/java9/module-info.java

```diff
@@ -0,0 +1,10 @@
+/***************************************************
+ * Licensed under MIT No Attribution (SPDX: MIT-0) *
+ ***************************************************/
+
+/**
+ * Reactive Streams
+ */
+module org.reactivestreams {
+  exports org.reactivestreams;
```

Comment:
The module name matches the previous `Automatic-Module-Name`. The exported package is the only package in the JAR, so export-related risk is minimal. If it needs to be an `open module` let me know.